docs/generators/dart-dio.md

```diff
@@ -3,11 +3,10 @@ title: Config Options for dart-dio
 sidebar_label: dart-dio
 ---
 
-These options may be applied as additional-properties (cli) or configOptions (plugins). Refer to [configuration docs](https://openapi-generator.tech/docs/configuration) for more details.
-
 | Option | Description | Values | Default |
 | ------ | ----------- | ------ | ------- |
 |allowUnicodeIdentifiers|boolean, toggles whether unicode identifiers are allowed in names or not, default is false| |false|
+|browserClient|Is the client browser based (for Dart 1.x only)| |null|
```

Comment:
dart1 has been removed now. looks like your branch isn't up to date with upstream master

Reply:
It is. Made sure of that just an hour ago. I'll check again.

Reply:
```shell
$ git status 
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
$ git fetch upstream master
From https://github.com/OpenAPITools/openapi-generator
 * branch                  master     -> FETCH_HEAD
$ git merge upstream/master
Already up to date.
$ git checkout refactor/dart2-template 
Switched to branch 'refactor/dart2-template'
Your branch is up to date with 'origin/refactor/dart2-template'.
$ git merge master
Already up to date.
```